docs/multi-user/admin/README.md

```diff
@@ -0,0 +1,113 @@
+# MinIO Admin Multi-user Quickstart Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+MinIO supports multiple admin users in addition to default root user created during server startup. New admins can be added after server starts up, and server can be configured to deny or allow access to different admin operations for these users. This document explains how to add/remove admin users and modify their access rights.
+
+
+## Get started
+In this document we will explain in detail on how to configure admin users.
+
+### 1. Prerequisites
+- Install mc - [MinIO Client Quickstart Guide](https://docs.min.io/docs/minio-client-quickstart-guide.html)
+- Install MinIO - [MinIO Quickstart Guide](https://docs.min.io/docs/minio-quickstart-guide)
+
+
+### 2. Create a new admin user with CreateUser, DeleteUser and ConfigUpdate permissions
+Use [`mc admin policy`](https://docs.min.io/docs/minio-admin-complete-guide.html#policies) to create custom admin policies.
+
+Create new canned policy file `adminManageUser.json`. This policy enables admin user to
+manage other users.
+```json
+cat > adminManageUser.json << EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "admin:CreateUser",
+        "admin:DeleteUser",
+        "admin:ConfigUpdate"
+      ],
+      "Effect": "Allow",
+      "Sid": ""
+    },
+    {
+      "Action": [
+        "s3:*"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:s3:::*"
+      ],
+      "Sid": ""
+    }
+  ]
+}
+EOF
+```
+
+Create new canned policy by name `adminManageUser` using `adminManageUser.json` policy file.
+```
+mc admin policy add myminio adminManageUser adminManageUser.json
+```
+
+Create a new admin user `admin1` on MinIO use `mc admin user`.
+```
+mc admin user add myminio admin1 admin123
+```
+
+Once the user is successfully created you can now apply the `adminManageUser` policy for this user.
+
+```
+mc admin policy set myminio adminManageUser user=admin1
+```
+
+This admin user will then be allowed to perform create/delete user operations via `mc admin user`
+
+### 3. Configure `mc` and create another user user1 with attached policy user1policy
+```
+mc config host add myminio-admin1 http://localhost:9000 admin1 admin123 --api s3v4
+
+mc admin user add myminio-admin1 user1 user123
+mc admin policy add myminio-admin1 user1policy ~/user1policy.json
+mc admin policy set myminio-admin1 user1policy user=user1
+```
+
```

Comment:
We should also mention that with these policies you can also externalize the admin users to IDP with special permissions